_data/apis.yml

```diff
@@ -334,6 +516,7 @@
 
 - name: Papago Translation
   id: papago
+  company: naver-cloud
```

Comment:
I think just Naver, similar to Google or Yandex

Reply:
My logic was the same as with IBM.